pkg/flow/syncer.go

```diff
@@ -1089,7 +1088,7 @@ func (repository *localRepository) clone(ctx context.Context) error {
 
 	if strings.HasPrefix(uri, prefix) && len(repository.repo.Passphrase) > 0 {
 		if !strings.Contains(uri, "@") {
-			uri = fmt.Sprintf("%s%s@", prefix, url.QueryEscape(repository.repo.Passphrase)) + strings.TrimPrefix(uri, prefix)
+			uri = fmt.Sprintf("%s%s@", prefix, repository.repo.Passphrase) + strings.TrimPrefix(uri, prefix)
```

Comment:
Has this change been well tested? I understand that it may improve compatibility with Gitlab. But I suspect that it will cause issues for passphrases containing characters that require url encoding.

Reply:
The tokens are generated by gitlab and github. we don't use username & password, only tokens. I mean, in theory you can add a token with user:password but it should not be done.